{
	"document": {
		"aggregate_severity": {
			"namespace": "https://access.redhat.com/security/updates/classification/",
			"text": "Important"
		},
		"category": "csaf_vex",
		"csaf_version": "2.0",
		"distribution": {
			"text": "Copyright © Red Hat, Inc. All rights reserved.",
			"tlp": {
				"label": "WHITE",
				"url": "https://www.first.org/tlp/"
			}
		},
		"lang": "en",
		"notes": [
			{
				"category": "summary",
				"text": "An update is now available for Red Hat Ansible Automation Platform 1.2\n\nRed Hat Product Security has rated this update as having a security impact of Important. A Common Vulnerability Scoring System (CVSS) base score, which gives a detailed severity rating, is available for each vulnerability from the CVE link(s) in the References section.",
				"title": "Topic"
			},
			{
				"category": "general",
				"text": "Red Hat Ansible Automation Platform integrates Red Hat’s automation suite consisting of Red Hat Ansible Tower, Red Hat Ansible Engine, and use-case specific capabilities for Microsoft Windows,network, security, and more, along with Software-as-a-Service (SaaS)-based capabilities and features for organization-wide effectiveness.\n\nSecurity Fix(es):\n\n* python3-django: Django: SQL injection in QuerySet.annotate(),aggregate() and extra() (CVE-2022-28346)\n\n* python3-django: Django: SQL injection via QuerySet.explain(options) on PostgreSQL (CVE-2022-28347)\n\nFor more details about the security issue(s), including the impact, a CVSS score, acknowledgments, and other related information, refer to the CVE page(s) listed in the References section.",
				"title": "Details"
			},
			{
				"category": "legal_disclaimer",
				"text": "This content is licensed under the Creative Commons Attribution 4.0 International License (https://creativecommons.org/licenses/by/4.0/). If you distribute this content, or a modified version of it, you must provide attribution to Red Hat Inc. and provide a link to the original.",
				"title": "Terms of Use"
			}
		],
		"publisher": {
			"category": "vendor",
			"contact_details": "https://access.redhat.com/security/team/contact/",
			"issuing_authority": "Red Hat Product Security is responsible for vulnerability handling across all Red Hat offerings.",
			"name": "Red Hat Product Security",
			"namespace": "https://www.redhat.com"
		},
		"references": [
			{
				"category": "self",
				"summary": "https://access.redhat.com/errata/RHSA-2022:5703",
				"url": "https://access.redhat.com/errata/RHSA-2022:5703"
			},
			{
				"category": "external",
				"summary": "https://access.redhat.com/security/updates/classification/#important",
				"url": "https://access.redhat.com/security/updates/classification/#important"
			},
			{
				"category": "external",
				"summary": "2072447",
				"url": "https://bugzilla.redhat.com/show_bug.cgi?id=2072447"
			},
			{
				"category": "external",
				"summary": "2072459",
				"url": "https://bugzilla.redhat.com/show_bug.cgi?id=2072459"
			},
			{
				"category": "self",
				"summary": "Canonical URL",
				"url": "https://access.redhat.com/security/data/csaf/v2/advisories/2022/rhsa-2022_5703.json"
			}
		],
		"title": "Red Hat Security Advisory: Red Hat Ansible Automation Platform 1.2 security update",
		"tracking": {
			"current_release_date": "2024-09-18T05:55:46+00:00",
			"generator": {
				"date": "2024-09-18T05:55:46+00:00",
				"engine": {
					"name": "Red Hat SDEngine",
					"version": "3.33.3"
				}
			},
			"id": "RHSA-2022:5703",
			"initial_release_date": "2022-07-25T19:53:29+00:00",
			"revision_history": [
				{
					"date": "2022-07-25T19:53:29+00:00",
					"number": "1",
					"summary": "Initial version"
				},
				{
					"date": "2022-07-25T19:53:29+00:00",
					"number": "2",
					"summary": "Last updated version"
				},
				{
					"date": "2024-09-18T05:55:46+00:00",
					"number": "3",
					"summary": "Last generated version"
				}
			],
			"status": "final",
			"version": "3"
		}
	},
	"product_tree": {
		"branches": [
			{
				"branches": [
					{
						"branches": [
							{
								"category": "product_name",
								"name": "Red Hat Automation Hub 4.2 for RHEL 8",
								"product": {
									"name": "Red Hat Automation Hub 4.2 for RHEL 8",
									"product_id": "8Base-Automation-Hub-4.2",
									"product_identification_helper": {
										"cpe": "cpe:/a:redhat:ansible_automation_platform:4.2::el8"
									}
								}
							},
							{
								"category": "product_name",
								"name": "Red Hat Automation Hub 4.2 for RHEL 7",
								"product": {
									"name": "Red Hat Automation Hub 4.2 for RHEL 7",
									"product_id": "7Server-Automation-Hub-4.2",
									"product_identification_helper": {
										"cpe": "cpe:/a:redhat:ansible_automation_platform:4.2::el7"
									}
								}
							}
						],
						"category": "product_family",
						"name": "Red Hat Ansible Automation Platform"
					},
					{
						"branches": [
							{
								"category": "product_version",
								"name": "python3-django-0:2.2.28-1.el8pc.src",
								"product": {
									"name": "python3-django-0:2.2.28-1.el8pc.src",
									"product_id": "python3-django-0:2.2.28-1.el8pc.src",
									"product_identification_helper": {
										"purl": "pkg:rpm/redhat/python3-django@2.2.28-1.el8pc?arch=src"
									}
								}
							},
							{
								"category": "product_version",
								"name": "python3-django-0:2.2.28-1.el7pc.src",
								"product": {
									"name": "python3-django-0:2.2.28-1.el7pc.src",
									"product_id": "python3-django-0:2.2.28-1.el7pc.src",
									"product_identification_helper": {
										"purl": "pkg:rpm/redhat/python3-django@2.2.28-1.el7pc?arch=src"
									}
								}
							}
						],
						"category": "architecture",
						"name": "src"
					},
					{
						"branches": [
							{
								"category": "product_version",
								"name": "python3-django-0:2.2.28-1.el8pc.noarch",
								"product": {
									"name": "python3-django-0:2.2.28-1.el8pc.noarch",
									"product_id": "python3-django-0:2.2.28-1.el8pc.noarch",
									"product_identification_helper": {
										"purl": "pkg:rpm/redhat/python3-django@2.2.28-1.el8pc?arch=noarch"
									}
								}
							},
							{
								"category": "product_version",
								"name": "python3-django-0:2.2.28-1.el7pc.noarch",
								"product": {
									"name": "python3-django-0:2.2.28-1.el7pc.noarch",
									"product_id": "python3-django-0:2.2.28-1.el7pc.noarch",
									"product_identification_helper": {
										"purl": "pkg:rpm/redhat/python3-django@2.2.28-1.el7pc?arch=noarch"
									}
								}
							}
						],
						"category": "architecture",
						"name": "noarch"
					}
				],
				"category": "vendor",
				"name": "Red Hat"
			}
		],
		"relationships": [
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python3-django-0:2.2.28-1.el7pc.noarch as a component of Red Hat Automation Hub 4.2 for RHEL 7",
					"product_id": "7Server-Automation-Hub-4.2:python3-django-0:2.2.28-1.el7pc.noarch"
				},
				"product_reference": "python3-django-0:2.2.28-1.el7pc.noarch",
				"relates_to_product_reference": "7Server-Automation-Hub-4.2"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python3-django-0:2.2.28-1.el7pc.src as a component of Red Hat Automation Hub 4.2 for RHEL 7",
					"product_id": "7Server-Automation-Hub-4.2:python3-django-0:2.2.28-1.el7pc.src"
				},
				"product_reference": "python3-django-0:2.2.28-1.el7pc.src",
				"relates_to_product_reference": "7Server-Automation-Hub-4.2"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python3-django-0:2.2.28-1.el8pc.noarch as a component of Red Hat Automation Hub 4.2 for RHEL 8",
					"product_id": "8Base-Automation-Hub-4.2:python3-django-0:2.2.28-1.el8pc.noarch"
				},
				"product_reference": "python3-django-0:2.2.28-1.el8pc.noarch",
				"relates_to_product_reference": "8Base-Automation-Hub-4.2"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python3-django-0:2.2.28-1.el8pc.src as a component of Red Hat Automation Hub 4.2 for RHEL 8",
					"product_id": "8Base-Automation-Hub-4.2:python3-django-0:2.2.28-1.el8pc.src"
				},
				"product_reference": "python3-django-0:2.2.28-1.el8pc.src",
				"relates_to_product_reference": "8Base-Automation-Hub-4.2"
			}
		]
	},
	"vulnerabilities": [
		{
			"cve": "CVE-2022-28346",
			"cwe": {
				"id": "CWE-89",
				"name": "Improper Neutralization of Special Elements used in an SQL Command ('SQL Injection')"
			},
			"discovery_date": "2022-04-06T00:00:00+00:00",
			"ids": [
				{
					"system_name": "Red Hat Bugzilla ID",
					"text": "2072447"
				}
			],
			"notes": [
				{
					"category": "description",
					"text": "A flaw was found in the Django package, which leads to a SQL injection. This flaw allows an attacker using a crafted dictionary containing malicious SQL queries to compromise the database completely.",
					"title": "Vulnerability description"
				},
				{
					"category": "summary",
					"text": "Django: SQL injection in QuerySet.annotate(),aggregate() and extra()",
					"title": "Vulnerability summary"
				},
				{
					"category": "other",
					"text": "Red Hat OpenStack does ship the affected version of Django, however, vulnerability is not exposed in the product as it does not make use of vulnerable code. We may update Django in a future release of OpenStack.",
					"title": "Statement"
				},
				{
					"category": "general",
					"text": "The CVSS score(s) listed for this vulnerability do not reflect the associated product's status, and are included for informational purposes to better understand the severity of this vulnerability.",
					"title": "CVSS score applicability"
				}
			],
			"product_status": {
				"fixed": [
					"7Server-Automation-Hub-4.2:python3-django-0:2.2.28-1.el7pc.noarch",
					"7Server-Automation-Hub-4.2:python3-django-0:2.2.28-1.el7pc.src",
					"8Base-Automation-Hub-4.2:python3-django-0:2.2.28-1.el8pc.noarch",
					"8Base-Automation-Hub-4.2:python3-django-0:2.2.28-1.el8pc.src"
				]
			},
			"references": [
				{
					"category": "self",
					"summary": "Canonical URL",
					"url": "https://access.redhat.com/security/cve/CVE-2022-28346"
				},
				{
					"category": "external",
					"summary": "RHBZ#2072447",
					"url": "https://bugzilla.redhat.com/show_bug.cgi?id=2072447"
				},
				{
					"category": "external",
					"summary": "https://www.cve.org/CVERecord?id=CVE-2022-28346",
					"url": "https://www.cve.org/CVERecord?id=CVE-2022-28346"
				},
				{
					"category": "external",
					"summary": "https://nvd.nist.gov/vuln/detail/CVE-2022-28346",
					"url": "https://nvd.nist.gov/vuln/detail/CVE-2022-28346"
				},
				{
					"category": "external",
					"summary": "https://www.djangoproject.com/weblog/2022/apr/11/security-releases/",
					"url": "https://www.djangoproject.com/weblog/2022/apr/11/security-releases/"
				}
			],
			"release_date": "2022-04-11T08:00:00+00:00",
			"remediations": [
				{
					"category": "vendor_fix",
					"details": "Before applying this update, make sure all previously released errata\nrelevant to your system have been applied.\n\nFor details on how to apply this update, refer to:\n\nhttps://access.redhat.com/articles/11258",
					"product_ids": [
						"7Server-Automation-Hub-4.2:python3-django-0:2.2.28-1.el7pc.noarch",
						"7Server-Automation-Hub-4.2:python3-django-0:2.2.28-1.el7pc.src",
						"8Base-Automation-Hub-4.2:python3-django-0:2.2.28-1.el8pc.noarch",
						"8Base-Automation-Hub-4.2:python3-django-0:2.2.28-1.el8pc.src"
					],
					"restart_required": {
						"category": "none"
					},
					"url": "https://access.redhat.com/errata/RHSA-2022:5703"
				}
			],
			"scores": [
				{
					"cvss_v3": {
						"attackComplexity": "LOW",
						"attackVector": "NETWORK",
						"availabilityImpact": "LOW",
						"baseScore": 9.4,
						"baseSeverity": "CRITICAL",
						"confidentialityImpact": "HIGH",
						"integrityImpact": "HIGH",
						"privilegesRequired": "NONE",
						"scope": "UNCHANGED",
						"userInteraction": "NONE",
						"vectorString": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:H/A:L",
						"version": "3.1"
					},
					"products": [
						"7Server-Automation-Hub-4.2:python3-django-0:2.2.28-1.el7pc.noarch",
						"7Server-Automation-Hub-4.2:python3-django-0:2.2.28-1.el7pc.src",
						"8Base-Automation-Hub-4.2:python3-django-0:2.2.28-1.el8pc.noarch",
						"8Base-Automation-Hub-4.2:python3-django-0:2.2.28-1.el8pc.src"
					]
				}
			],
			"threats": [
				{
					"category": "impact",
					"details": "Important"
				}
			],
			"title": "Django: SQL injection in QuerySet.annotate(),aggregate() and extra()"
		},
		{
			"cve": "CVE-2022-28347",
			"cwe": {
				"id": "CWE-89",
				"name": "Improper Neutralization of Special Elements used in an SQL Command ('SQL Injection')"
			},
			"discovery_date": "2022-04-06T00:00:00+00:00",
			"ids": [
				{
					"system_name": "Red Hat Bugzilla ID",
					"text": "2072459"
				}
			],
			"notes": [
				{
					"category": "description",
					"text": "A flaw was found in the Django package, leading to a SQL injection. This flaw allows an attacker using a crafted dictionary containing malicious SQL queries to compromise the database completely.",
					"title": "Vulnerability description"
				},
				{
					"category": "summary",
					"text": "Django: SQL injection via QuerySet.explain(options) on PostgreSQL",
					"title": "Vulnerability summary"
				},
				{
					"category": "other",
					"text": "Red Hat OpenStack does ship the affected version of Django. However, the product is not vulnerable since it does not implement the vulnerable method QuerySet.explain() introduced in Django 2.1.x onward.",
					"title": "Statement"
				},
				{
					"category": "general",
					"text": "The CVSS score(s) listed for this vulnerability do not reflect the associated product's status, and are included for informational purposes to better understand the severity of this vulnerability.",
					"title": "CVSS score applicability"
				}
			],
			"product_status": {
				"fixed": [
					"7Server-Automation-Hub-4.2:python3-django-0:2.2.28-1.el7pc.noarch",
					"7Server-Automation-Hub-4.2:python3-django-0:2.2.28-1.el7pc.src",
					"8Base-Automation-Hub-4.2:python3-django-0:2.2.28-1.el8pc.noarch",
					"8Base-Automation-Hub-4.2:python3-django-0:2.2.28-1.el8pc.src"
				]
			},
			"references": [
				{
					"category": "self",
					"summary": "Canonical URL",
					"url": "https://access.redhat.com/security/cve/CVE-2022-28347"
				},
				{
					"category": "external",
					"summary": "RHBZ#2072459",
					"url": "https://bugzilla.redhat.com/show_bug.cgi?id=2072459"
				},
				{
					"category": "external",
					"summary": "https://www.cve.org/CVERecord?id=CVE-2022-28347",
					"url": "https://www.cve.org/CVERecord?id=CVE-2022-28347"
				},
				{
					"category": "external",
					"summary": "https://nvd.nist.gov/vuln/detail/CVE-2022-28347",
					"url": "https://nvd.nist.gov/vuln/detail/CVE-2022-28347"
				},
				{
					"category": "external",
					"summary": "https://www.djangoproject.com/weblog/2022/apr/11/security-releases/",
					"url": "https://www.djangoproject.com/weblog/2022/apr/11/security-releases/"
				}
			],
			"release_date": "2022-04-11T08:00:00+00:00",
			"remediations": [
				{
					"category": "vendor_fix",
					"details": "Before applying this update, make sure all previously released errata\nrelevant to your system have been applied.\n\nFor details on how to apply this update, refer to:\n\nhttps://access.redhat.com/articles/11258",
					"product_ids": [
						"7Server-Automation-Hub-4.2:python3-django-0:2.2.28-1.el7pc.noarch",
						"7Server-Automation-Hub-4.2:python3-django-0:2.2.28-1.el7pc.src",
						"8Base-Automation-Hub-4.2:python3-django-0:2.2.28-1.el8pc.noarch",
						"8Base-Automation-Hub-4.2:python3-django-0:2.2.28-1.el8pc.src"
					],
					"restart_required": {
						"category": "none"
					},
					"url": "https://access.redhat.com/errata/RHSA-2022:5703"
				}
			],
			"scores": [
				{
					"cvss_v3": {
						"attackComplexity": "LOW",
						"attackVector": "NETWORK",
						"availabilityImpact": "LOW",
						"baseScore": 9.4,
						"baseSeverity": "CRITICAL",
						"confidentialityImpact": "HIGH",
						"integrityImpact": "HIGH",
						"privilegesRequired": "NONE",
						"scope": "UNCHANGED",
						"userInteraction": "NONE",
						"vectorString": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:H/A:L",
						"version": "3.1"
					},
					"products": [
						"7Server-Automation-Hub-4.2:python3-django-0:2.2.28-1.el7pc.noarch",
						"7Server-Automation-Hub-4.2:python3-django-0:2.2.28-1.el7pc.src",
						"8Base-Automation-Hub-4.2:python3-django-0:2.2.28-1.el8pc.noarch",
						"8Base-Automation-Hub-4.2:python3-django-0:2.2.28-1.el8pc.src"
					]
				}
			],
			"threats": [
				{
					"category": "impact",
					"details": "Important"
				}
			],
			"title": "Django: SQL injection via QuerySet.explain(options) on PostgreSQL"
		}
	]
}
